cpp/src/aztec3/circuits/kernel/private/.test.cpp

```diff
@@ -409,20 +364,15 @@ TEST(private_kernel_tests, test_native_deposit)
 
     Prover mock_kernel_prover = mock_kernel_composer.create_prover();
     NT::Proof mock_kernel_proof = mock_kernel_prover.construct_proof();
+    // info("\nmock_kernel_proof: ", mock_kernel_proof.proof_data);
 
     std::shared_ptr<NT::VK> mock_kernel_vk = mock_kernel_composer.compute_verification_key();
```

Comment:
@iAmMichaelConnor this is the big chunk we should be able to remove once we have the special kernel version for the initial iteration. Won't need to compute the mock kernel anymore.